defect-tracker-server/src/main/resources/application-dev.properties

```diff
@@ -1,4 +1,4 @@
-server.port=1724
+server.port=1725
```

Comment:
no need to add application-dev properties file

Reply:
okay no add next time